tests/python/direct/test_python_frontend.py

```diff
@@ -575,12 +577,12 @@ def fusion_func(fd: FusionDefinition):
         t1 = fd.from_pytorch(inputs[1])
 
         t0_b = fd.ops.expand(t0, inputs[1].size())
-        t2 = fd.ops.add(t0_b, t1)
+        t2 = fd.ops.mul(t0_b, t1)
```

Comment:
Changed from `add` to `mul` to avoid conflict with `test_explicit_broadcast_input` which is caused by updates to `broadcast_in_dim` lowering.